### PR TITLE
added summary step

### PR DESCRIPTION
**Pull request by sirLisko**
**What**  
- added summary step to the forms

Screenshot 2020-11-17 at 12 15 28

**How to test it**
- check the `adult-referral` form
- after the last step there will be the Summary
- the submission has to work as before

### CODE REVIEW

**Comment by naomigreen** [comment]
Looks great! Just confirming the best method to get back to the summary page.

File: components/Summary/Summary.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            ? Object.values(formData[name]).filter(Boolean).map(MultiValue)
+            : typeof formData[name] === 'boolean'
+            ? JSON.stringify(formData[name])
+            : formData[name],
+        }))}
+    />
+  );
+  return (
+    <div className="govuk-!-margin-bottom-7">
+      <h2>{title}</h2>
```
Comment (naomigreen) [inline components/Summary/Summary.jsx:44]: Could you add the govuk-fieldset__legend--m className, so we get the correct font

Comment (sirLisko) [reply components/Summary/Summary.jsx:44]: Good idea, but it's not a `fieldset`, let me see if I can find something similar 🤔

Comment (sirLisko) [reply components/Summary/Summary.jsx]: I am going to use `govuk-heading-m and making the element `h3` so it's more semantic 👍

**Review by naomigreen — APPROVED** [review]

**Pull request merged**